libs/design-system/tree/src/lib/tree-styles/tree-styles.component.scss

```diff
@@ -0,0 +1,62 @@
+mat-tree {
+  --mat-tree-node-text-font: 'Metropolis';
```

Comment:
The font values should come from variables. Set the value to `var(--sys-label-medium-font)` and `var(--sys-label-medium-weight)` for the weight